libs/constants/src/lib/health-care-services.ts

```diff
@@ -0,0 +1,6 @@
+export const HEALTH_CARE_SERVICES = {
```

Comment:
Consider adding a folder for similar static data and put this data there as json. Later we can add all such static data files there.

Reply:
Notice it is in the `constants` library. All constants go there (right now there are cities and health care services).

Reply:
Both cities and health care services are not constants, but static data. Except for special cases, you never use cities.Haifa in your code.
But not critical at this stage of the project.

Reply:
They are constants at the moment. They are more like ENUMs actually, but constants was what we went for.
I'm not using CITIES[148] because I don't need to. It's just so BE and FE can talk with numbers instead of strings but show something meaningful to the user (a list of city strings).
I WILL use `CITIES[148]` when I implement the multiple choice of cities for filter (as shown in the filter interface.